### PR TITLE
Week 2 Step 3

### PR DESCRIPTION
**Pull request by anamatasanch**
- Add Gson dependency to POM
- Modify DataServlet to contain a comments ArrayList and return comments JSON
- Modify script.js to fetch the JSON string from the server and add them to the comment section
- Add comment section -> 
CommentsSec

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,19 +14,35 @@
...
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Ana!");
+    ArrayList<String> comments = new ArrayList<String>();
+    comments.add("Hello");
+    comments.add("How are you?");
+    comments.add("Greetings");
```
Comment (joanapascoal) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:34]: Optional: This array can be out of the doGet method, that way it is accessible by all the class methods. Which it isn't fundamental in this case.
`private ArrayList<String> comments = new ArrayList<String>();`

**Review by joanapascoal — APPROVED** [review]
Looks great!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,19 +14,35 @@
...
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Ana!");
+    ArrayList<String> comments = new ArrayList<String>();
```
Comment (TsueiCat) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:31]: you can initialize with just new ArrayList<>() with type inference :)

**Review by TsueiCat — APPROVED** [review]
Nice work!

**Pull request merged**